packages/ui/react-ui-stack/src/components/Section.tsx

```diff
@@ -28,6 +28,8 @@ export type SectionProps = PropsWithChildren<{
   // Data props.
   id: string;
   title: string;
+  index: number;
+  count: number;
```

Comment:
Prefer using tailwind’s `first` and `last` selectors over passing props purely for stylistic reasons: https://tailwindcss.com/docs/hover-focus-and-other-states#first

Reply:
nice. i was able to use group and group-first:border-be, etc.